linguatec_lexicon_frontend/views.py

```diff
@@ -140,15 +170,41 @@ class PrivacyPolicy(LinguatecBaseView):
 class SearchView(LinguatecBaseView):
     template_name = "linguatec_lexicon_frontend/search_results.html"
 
+
+    def order_lexicons(self, src_languages, lexicons):
+        res=[]
+        count=0
+        for source_language in src_languages:
+            res.append([])
+            for lex in lexicons:
+                if source_language == lex.get('src_language'):
+                    res[count].append(lex)
+            count=+1
+        return res
+
```

Comment:
Seems that it duplicates code of lines 123 to 141, doesn't it?

Reply:
Refactored to implement this method only once